sdcm/cluster.py

```diff
@@ -1828,6 +1830,10 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
         self.remoter.run('sudo mv /tmp/scylla.yaml {}'.format(yaml_file))
 
         if append_scylla_args:
+            scylla_help = self.remoter.run("scylla --help", ignore_status=True).stdout
+            scylla_arg_parser = ScyllaArgParser.from_scylla_help(scylla_help)
+            append_scylla_args = scylla_arg_parser.filter_args(append_scylla_args)
```

Comment:
Let's add info/debug message printing the args that were selected.

Reply:
done